installer/PowerToysSetup/Product.wxs

```diff
@@ -860,7 +860,7 @@
         <DirectoryRef Id="FileExplorerPreviewInstallFolder" FileSource="$(var.RepoDir)\modules\FileExplorerPreview\">
             <?foreach File in $(var.PowerPreviewFiles)?>
                 <Component Id="Module_PowerPreview_$(var.File)" Win64="yes">
-                    <File Id="$(var.File)" Source="$(var.BinX64Dir)modules\FileExplorerPreview\$(var.File)" />
+                    <File Id="Module_PowerPreview_$(var.File)" Source="$(var.BinX64Dir)modules\FileExplorerPreview\$(var.File)" />
```

Comment:
@yuyoyuppe Can this break something? Without this change duplicate id error is reported by wix. Not sure why, is ID same as file name if ID is not set explicitly.

Reply:
we can't have multiple IDs on a global level, so it's a name collision. that change is harmless.